data/json/recipes/recipe_medsandchemicals.json

```
@@ -1580,5 +1580,45 @@
     "qualities": [ { "id": "CONTAIN", "level": 1 } ],
     "tools": [ [ [ "syringe", -1 ] ], [ [ "surface_heat", 1, "LIST" ] ] ],
     "components": [ [ [ "hi_q_distillate", 10 ] ], [ [ "element", 1 ] ], [ [ "plastic_chunk", 1 ] ], [ [ "glass_shard", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "formic_acid",
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_DRUGS",
+    "skill_used": "chemistry",
+    "difficulty": 2,
+    "time": "20 m",
+    "qualities": [ { "id": "CONTAIN", "level": 1 } ],
+    "autolearn": true,
+    "//": "Should likely come from an entomology text, or something similar, in the future; but still relatively basic.",
+    "tools": [ [ [ "mortar_pestle", -1 ], [ "funnel", -1 ] ] ],
+    "components": [ [ [ "wasp_sting", 1 ], [ "bee_sting", 1 ] ] ]
+  },
+  {
+  	"type": "recipe",
+  	"activity_level": "LIGHT_EXERCISE",
+  	"result": "latex",
+  	"category": "CC_CHEM",
+  	"subcategory": "CSC_CHEM_OTHER",
+  	"skill_used": "chemistry",
+  	"difficulty": 2,
+  	"time": "3 m",
+  	"qualities": [ { "id": "CONTAIN", "level": 1 } ],
+  	"tools": [ [ [ "mortar_pestle", -1 ] ] ],
+  	"components": [ [ [ "spurge", 1 ] ] ]
```

```suggestion
  	"components": [ [ [ "spurge", 20 ] ] ]
```

Love everything about this, but you aren't getting two cups of latex put of one flower

---

Ha, fair point. Spurges are freakin' everywhere anyway.